test-example/dev-test.strest.ts

```diff
@@ -18,11 +18,10 @@ export class LoginStressTest extends MeteorStressTest {
   createStress(utils: SetupUtils) {
     utils.startTime();
 
-    return this.connect('http://localhost:3000').then(() => {
-      utils.stopTime();
-
-      return this.ddp;
-    });
+    return this.connect('http://localhost:3000')
+               .then(() => this.login('admin', null, '123456'))
+               .then(() => utils.stopTime())
```

Comment:
I am thinking about return a Promise that only chains the value it gets from `utils.stopTime()` and `utils.startTime`, that way you can use it that way:
`.then(utils.stopTime)`. 

Also, we no longer need to return `this.ddp` in the end of the chain, because each test instance has it's own context, so the DDPClient object is stored on `this` as `ddp` or `ddpClient` :)